x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -68,14 +69,22 @@ export const LatestFindingsContainer = ({ dataView }: FindingsBaseProps) => {
    */
   const findingsGroupByNone = useLatestFindings({
     ...getPaginationQuery({
-      pageIndex: urlQuery.pageIndex,
-      pageSize: pageSize || urlQuery.pageSize,
+      pageIndex: 0,
+      pageSize: MAX_ITEMS,
```

Comment:
since these values don't change now, we could instead inline the result of `getPaginationQuery` directly to `useLatestFindings`. as in - `from: 0, size: 500`